src/Graphics/OpenGLContext/opengl_BufferManipulationObjectFactory.cpp

```diff
@@ -81,6 +81,7 @@ class AddFramebufferTexture2D : public AddFramebufferRenderTarget
 
 	void addFrameBufferRenderTarget(const graphics::Context::FrameBufferRenderTarget & _params) override
 	{
+		gfxContext.bindFramebuffer(_params.bufferTarget, _params.bufferHandle);
```

Comment:
This is necessary to update the m_boundDrawFBO/m_boundReadFBO variables, there may be a better way?

The other option would be to not count on addFrameBufferRenderTarget to bind the FBO, and call gfx.bindFramebuffer() in the frontend before calling addFrameBufferRenderTarget. The same goes for blitFramebuffers